rest-api/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"net/http"
+
+	controller "./controllers"
```

Comment:
Try to use go modules in your project in order to avoid the use of relative paths

Reply:
This one has been fixed in my last commit